lerna.json

```diff
@@ -4,7 +4,7 @@
   "version": "independent",
   "command": {
     "publish": {
-      "ignoreChanges": [
+      "ignore": [
```

Comment:
I found that there was a lerna warning of this being deprecated.

Reply:
Ah interesting, what command did you run to see this warning?